sql/parse/parse_test.go

```diff
@@ -1507,6 +1507,26 @@ var fixtures = map[string]sql.Node{
 		`SELECT * FROM foo`,
 		true,
 	),
+	`SELECT 2 UNION SELECT 3` : plan.NewUnion(
```

Comment:
Probably should add a couple tests for distinct, as well as unions of 3 or more select statements